_i18n/en.yml

```
@@ -240,6 +240,8 @@ press-kit:
   dontbuystickerp: Spread Monero everywhere with the help of this sticker. Available in multiple languages and formats (vectors included).
   guerrillakit: Guerrilla Toolkit
   guerrillakitp: A document created by the Monero Outreach workgroup containing materials and tips for an effective guerrilla marketing campaign.
+  promographics: "Monero Promotional graphics : Badges and stickers for websites"
```

```suggestion
  promographics: "Monero Promotional graphics - Badges and stickers for websites"
```

A stray `:` looks weird.

---

Fixed